pytrip/tripexecuter/kernel.py

```diff
@@ -25,7 +25,6 @@ class KernelModel(object):
     """
     Kernel data, holding all necessary data for a certain projectile
     """
-
```

Comment:
@ljelen Can you avoid meaningless white-space changes ? The best is to remove using Pycharm "Compare with master branch" and restoring the file to its status on master.

Reply:
There is no new line:
https://github.com/pytrip/pytrip/blob/b774d3572663c68a04c1936a91f115f963828233/pytrip/ddd.py#L33
There is new line:
https://github.com/pytrip/pytrip/blob/b774d3572663c68a04c1936a91f115f963828233/pytrip/volhist.py#L32

I know it doesn't matter but I would prefer to stick to one convention.

Reply:
ok